controllers/storageclassclaim_migration_controller.go

```diff
@@ -82,7 +82,7 @@ func (r *StorageClassClaimMigrationReconciler) Reconcile(ctx context.Context, re
 
 	storageClaim.Spec.EncryptionMethod = storageClassClaim.Spec.EncryptionMethod
 	storageClaim.Spec.StorageProfile = storageClassClaim.Spec.StorageProfile
-	storageClaim.Spec.StorageClient = storageClassClaim.Spec.StorageClient.DeepCopy()
```

Comment:
What about name collision? shouldn't we compose a new name based on both the namespace and the name?

Reply:
I believe this whole migration controller should go away.

Reply:
I don't have any info when this migratiom controller is useful.

Simply migrating the CR from storageclassclaim to storageclaim may create new resources (rns & svg) which are fundamentally diff and so why can't the migration be carried out during at one place ie, migration script.

Reply:
Ok. Can you open an issue about the necessity/lack of necessity of the entire migration controller? link it here and I will resolve this.

Reply:
done, https://github.com/red-hat-storage/ocs-client-operator/issues/135